plugins/tailwind/compiler.ts

```diff
@@ -62,6 +68,39 @@ export async function initTailwind(
     return pattern;
   });
 
+  let importMap;
+  if (path.extname(config.denoJsonPath) === ".json") {
+    importMap = (await import(path.toFileUrl(config.denoJsonPath).href, {
+      with: { type: "json" },
+    })).default;
+  } else if (path.extname(config.denoJsonPath) === ".jsonc") {
+    const fileContents = Deno.readTextFileSync(config.denoJsonPath);
+    importMap = jsoncParse(fileContents);
+  } else {
+    throw Error("deno config must be either .json or .jsonc");
+  }
```

Comment:
@iuioiua, are you sure there's no room in the std library for something like this? I've just realized this won't work if the user is using `importMap` instead of `imports` in their `deno.json(c)` file. Surely I'm not the only person who wants to read the config file and get a certain property off of it.

Ideally this entire block could be condensed to:
```ts
const importMap = await readConfig(config.denoJsonPath).imports();
```

`readConfig` would take care of handling `.json` and `.jsonc` differences, and `importMap()` would handle whether it's in the config itself, or buried in a separate import map.

Reply:
I ended up getting something like this working:
```ts
const importMap = getImportMap(readConfig(config.denoJsonPath));
```

Let me know if I should create a PR in `deno_std` to add this.

Reply:
Hm... That actually might be good for the Deno ecosystem. Can you open an issue with your proposal?

Reply:
Sure, I'll open one tomorrow.